src/SwipeProfiles.js

```diff
@@ -150,48 +156,53 @@ class SwipeProfiles extends React.Component {
     return (
       /*navigation bar and other necessary information about the match*/
       <div className="SwipeProfile">
-        <Header id={this.state.id}/>
+        <Header id={this.state.id} />
         <br></br>
         <header class="pageTitle">Potential Friends!</header>
```

Comment:
Move this nav bar to its own separate div or it will be effected by the "swipeprofile" styling. Move it to before the div above.

Reply:
Please make sure your other pages does this too. It's to ensure the nav bar sticks to the top of all the pages.

Reply:
theres no navbar code here?

Reply:
the header id = this.state.id is the nav bar (its calling the Header file) just move it before the <div className = "swipeProfile"> into its own div